src/connector/Connector.ts

```diff
@@ -1,3 +1,9 @@
+/**
```

Comment:
The connector is to be used by the other object (provider and account).

You should explain that because the documentation will target two kind of devs:
- the one using our library and they won't use this object
- the developer that want to work on this repo and they will be interested on this documentation